.gitignore

```diff
@@ -40,3 +40,5 @@ TestResult.xml
 *.err
 GlobalAssemblyInfo.cs
 CommonAssemblyInfo.cs
+All Tests in CorrugatedIron.dcvr.settings
```

Comment:
`All Tests in CorrugatedIron.dcvr.settings` doesn't appear to be a valid file name ... is it?